ssl/ssl_conf.c

```diff
@@ -318,7 +319,8 @@ static int protocol_from_string(const char *value)
         {"TLSv1.2", TLS1_2_VERSION},
         {"TLSv1.3", TLS1_3_VERSION},
         {"DTLSv1", DTLS1_VERSION},
-        {"DTLSv1.2", DTLS1_2_VERSION}
+        {"DTLSv1.2", DTLS1_2_VERSION},
+        {"DTLSv1.3", DTLS1_3_VERSION}
```

Comment:
Probably there needs to be an equivalent update to doc/man3/SSL_CONF_cmd.pod

Reply:
I've pushed some changes in another pr: https://github.com/openssl/openssl/pull/22363/files#diff-158c83a80afb8261ce0db074e9a2e6991a9b487bc63974bed7bc2e9c7450e4d7
That fixes this.

Please let me know if you think it is better to add the relevant documentation changes to this pr.

Reply:
In general I'd prefer to keep the documentation changes inside the PR that makes the corresponding change. But that's not always practical and since you've already split things up in this way I'm happy to continue with that.

Note though that I don't think that the commit you reference quite covers it. In particular the MaxProtocol and MinProtocol sections on the SSL_CONF_cmd.pod page don't mention DTLSv1.3.

Reply:
I'll go through SSL_CONF_cmd.pod in the other PR then, but remember for future changes to make the corresponding doc change in the same PR. 

What is your opinion on changing occurences of TLSv1.3 with (D)TLSv1.3 for the documentation that applies to both TLS and DTLS protocols? 

For example:
```
=item B<-ciphersuites> I<1.3ciphers>

Sets the available ciphersuites for TLSv1.3 to value. This is a
colon-separated list of TLSv1.3 ciphersuite names in order of preference. This
list will be combined any configured TLSv1.2 and below ciphersuites.
See L<openssl-ciphers(1)> for more information.
```
becomes
```
=item B<-ciphersuites> I<1.3ciphers>

Sets the available ciphersuites for (D)TLSv1.3 to value. This is a
colon-separated list of (D)TLSv1.3 ciphersuite names in order of preference. This
list will be combined any configured (D)TLSv1.2 and below ciphersuites.
See L<openssl-ciphers(1)> for more information.
```
?

Reply:
Yes, I think `(D)TLSv1.3` is the correct way to show this.

Reply:
Cool I'll make the update to the documentation pull request.